pkg/volume/local/local_test.go

```diff
@@ -57,6 +60,44 @@ func getPlugin(t *testing.T) (string, volume.VolumePlugin) {
 	return tmpDir, plug
 }
 
+func getBlockPlugin(t *testing.T) (string, volume.BlockVolumePlugin) {
+	tmpDir, err := utiltesting.MkTmpdir("localVolumeTest")
+	if err != nil {
+		t.Fatalf("can't make a temp dir: %v", err)
+	}
+	fakeDev := path.Join(tmpDir, testDev)
+	file, err := os.OpenFile(fakeDev, os.O_RDONLY|os.O_CREATE, 0666)
+	if err != nil {
+		t.Fatalf("Cannot create fakedev: %v", err)
+	}
+	err = file.Close()
+	if err != nil {
+		t.Fatalf("Cannot close fakedev: %v", err)
+	}
+	podPath := path.Join(tmpDir, testPodPath)
```

Comment:
Do you actually need to create/symlink the pod path here?  The plugin itself doesn't actually do anything except pass through the device paths.